airflow/providers/google/marketing_platform/example_dags/example_google_ads.py

```diff
@@ -0,0 +1,83 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""
+Example Airflow DAG that shows how to use GoogleAdsToGcsOperator.
+"""
+import os
+
+from airflow import models
+from airflow.providers.google.marketing_platform.operators.google_ads import GoogleAdsToGcsOperator
+from airflow.utils import dates
+
+# [START howto_google_ads_env_variables]
+CLIENT_IDS = ["7314394221", "9725033159"]
```

Comment:
Can those values be public? :)

Reply:
@nuclearpinguin They are just random number that I generated, so no concern from my end.

Reply:
I can replace with something even more random (e.g. 1111111, 2222222) to expressly display that they are fake.